src/components/package-show.js

```diff
@@ -22,100 +20,96 @@ export default function PackageShow({ vendorPackage, packageTitles }, { intl })
   };
 
   return (
-    <div data-test-eholdings-package-details>
-      <Paneset>
-        <Pane defaultWidth="100%">
-          {vendorPackage.isResolved ? (
-            <div>
-              <div style={{ margin: '2rem 0' }}>
-                <KeyValueLabel label="Package">
-                  <h1 data-test-eholdings-package-details-name>
-                    {packageRecord.packageName}
-                  </h1>
-                </KeyValueLabel>
+    <div style={{ padding: '0 1rem' }} data-test-eholdings-package-details>
```

Comment:
i'm not the most proficient CSS person, but aren't inline styles generally frowned upon?  or are there certain cases where using them is best practice?

Reply:
You're right, but there are already inline styles in this component, and so we came to the conclusion that most of the styling will need some polish anyway

Reply:
Certainly should not be permanent.